diff --git a/pyproject.toml b/pyproject.toml
index 654783b78c..331fcc6860 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -65,13 +65,13 @@ mcp = {version = "1.26.0", optional = true, python = ">=3.10"}
 a2a-sdk = {version = "0.3.25", optional = true, python = ">=3.10"}
 litellm-proxy-extras = {version = "0.4.65", optional = true}
 rich = {version = "13.9.4", optional = true}
-litellm-enterprise = {version = "0.1.37", optional = true}
+litellm-enterprise = {version = "0.1.37", optional = true, markers = "platform_machine != 'ppc64le'"}
 diskcache = {version = "5.6.3", optional = true}
-polars = {version = "1.39.3", optional = true, python = ">=3.10"}
+polars = {version = "1.39.3", optional = true, python = ">=3.10", markers = "platform_machine != 'ppc64le'"}
 semantic-router = {version = "0.1.12", optional = true, python = ">=3.9,<3.14"}
 mlflow = {version = "3.9.0", optional = true, python = ">=3.10"} # not in PyPI proxy extra
 soundfile = {version = "0.12.1", optional = true}
-pyroscope-io = {version = "0.8.16", optional = true, markers = "sys_platform != 'win32'"}
+pyroscope-io = {version = "0.8.16", optional = true, markers = "sys_platform != 'win32' and platform_machine != 'ppc64le'"}
 # grpcio: pinned to 1.80.0 (past reconnect bug #38290 in 1.68.x, has Python 3.14 wheels)
 grpcio = {version = "1.80.0", optional = true}
